jupyter-config/cern-config/analytix.sh

```diff
@@ -0,0 +1,2 @@
+export SPARK_DEMO_OPTION_CLUSTER_NAME=analytix
```

Comment:
Is this to source the configuration related to analytix? Why DEMO?

Reply:
That was a development-only script intended to check if the pre_script option works. I've removed it now :)